frontend/src/app/password-reset/password-reset-confirm.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+
+import { AuthService } from '../auth.service';
+
+@Component({
+  selector: 'volontulo-password-reset-confirm',
+  templateUrl: './password-reset-confirm.component.html',
+  styleUrls: ['./password-reset.component.scss'],
+})
+export class PasswordResetConfirmComponent implements OnInit {
+  resetForm: FormGroup;
+  uidb64: string;
+  token: string;
+
+  constructor(private activatedRoute: ActivatedRoute,
+              private authService: AuthService,
+              private router: Router,
+              private fb: FormBuilder) {}
+
+  ngOnInit() {
+    this.activatedRoute.params
+      .subscribe(params => {
+        this.uidb64 = params.uidb64;
+        this.token = params.token;
+      });
+    this.resetForm = this.fb.group({
+      'passwords': this.fb.group({
+        'password': this.fb.control(null, Validators.required),
+        'confirmPassword': this.fb.control(null, Validators.required),
+      }, {validator: this.checkPasswords})
+    });
+  }
+
+  checkPasswords(group: FormGroup): {[key: string]: boolean} {
+    const password = group.get('password').value;
+    const confirmPassword = group.get('confirmPassword').value;
+    return password === confirmPassword ? null : { notEqual: true }
+  }
+
+  onSubmit(): void {
+    const password = this.resetForm.get('passwords.password').value;
+    this.authService.confirmResetPassword({password}, this.uidb64, this.token)
```

Comment:
If you are not planning to extend {password} with more properties, then use a string as a parameter, do not wrap it in object.